ci/jenkins/pipelines/skuba-upgrade-nightly.Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+/**
+ * This pipeline verifies skuba node/cluster upgrade plan/apply
+ */
+
+pipeline {
+   agent { node { label 'caasp-team-private' } }
+
+   environment {
+        OPENSTACK_OPENRC = credentials('ecp-openrc')
+        TERRAFORM_STACK_NAME = "${JOB_NAME}-${BUILD_NUMBER}"
+        GITHUB_TOKEN = credentials('github-token')
+        VMWARE_ENV_FILE = credentials('vmware-env')
+   }
+
+   stages {
+        stage('Getting Ready For Cluster Deployment') { steps {
+            sh(script: 'make -f skuba/ci/Makefile pre_deployment', label: 'Pre Deployment')
+        } }
+
+        parallel {
```

Comment:
would it create 2 stacks with the same name ("`${JOB_NAME}-${BUILD_NUMBER}`") at the same time? maybe you'd have to pass this as an `env` to each test and remove the global `TERRAFORM_STACK_NAME` env?

Reply:
Done!